src/swarmus_ros_simulation/launch/slam_from_cell.launch

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" ?>
+<launch>
+    <include file="$(find swarmus_ros_navigation)/launch/navigation.launch"/>
+    <include file="$(find swarmus_ros_simulation)/launch/slam_simulation.launch"/>
+
+    <node pkg="hiveboard_bridge" type="hiveboard_bridge" name="hiveboard_bridge" output="screen">
+        <param name="TCP_SERVER_PORT" type="int" value="8080"/>
+        <param name="ROBOT_NAME" type="str" value="pioneer_0"/>
```

Comment:
Since you are using the `pioneer_0` at line 8 and since the `navigation.launch` and the `slam_simulation.launch` have a `robot_name` argument , I would add another `robot_name` in this file that will set the `robot_name` of the others. With that, we will able to change the `robot_name` at one place only.